release/src/main/scripts/deploy_pypi.sh

```diff
@@ -41,9 +41,12 @@ fi
 mkdir ${LOCAL_CLONE_DIR}
 cd ${LOCAL_CLONE_DIR}
 
+virtualenv deploy_pypi_env
+source ./deploy_pypi_env/bin/activate
+pip install twine
```

Comment:
Consider making this in a /tmp directory or the directory that you create and later clean up (assuming it won't interfere with pypi uploads).

Reply:
The virtual env is created in `LOCAL_CLONE_DIR`. It won't interfere because we download artifacts to `LOCAL_CLONE_DIR/dist.apache.org`.